### PR TITLE
Improve scripts to end when subcommand fails

### PR DESCRIPTION
**Pull request by Michkov**
- exit script when subcommand fails
- improve text output

### CODE REVIEW

File: hack/flags.sh [hunk 1]
```diff
@@ -59,6 +59,9 @@ parse_flags() {
 
   if [ "${MODE}" == "preview" ]
   then
+    if [ "${PARSE_FLAGS_RUN}" == "1" ]; then
+      return
```
Comment (MatousJobanek) [inline hack/flags.sh:63]: could you add a comment to explain why it returns - it doesn't have to be obvious for the first glance.

File: hack/install-ckcp.sh [hunk 2]
```diff
@@ -18,19 +18,27 @@ if [ "$1" == "--destroy" ]; then
   oc delete project spi-vault
 fi
 
+set -e
+
```
Comment (MatousJobanek) [inline hack/install-ckcp.sh:22]: we could move it to the very top of the script

File: hack/util-patch-spi-config.sh [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e
```
Comment (MatousJobanek) [inline hack/util-patch-spi-config.sh:1]: let's keep the scripts consistent - either use this one or the `set -e` one

File: hack/install-ckcp.sh (hunk 2 above)
Comment (Michkov) [reply hack/install-ckcp.sh]: moved and disabled for destroy part

File: hack/util-patch-spi-config.sh (hunk 3 above)
Comment (Michkov) [reply hack/util-patch-spi-config.sh]: updated

File: hack/util-set-github-org [hunk 4]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e
```
Comment (MatousJobanek) [inline hack/util-set-github-org:1]:
```suggestion
#!/bin/bash

set -e
```

**Review by MatousJobanek — APPROVED** [review]

**Pull request merged**